pa11y_scripts/.pa11y-ci-p4.json

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
This is great, let's add it to circle here https://github.com/usdoj-crt/crt-portal/blob/a3790991ce73a8cbea7f3b895767321390a8f7ed/.circleci/config.yml#L66

Reply:
Another great catch, thanks!